src/Command/AddTerritorialCouncilMemberInfoCommand.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\Adherent;
+use App\Entity\TerritorialCouncil\TerritorialCouncilMembership;
+use App\Entity\TerritorialCouncil\TerritorialCouncilQuality;
+use App\Entity\TerritorialCouncil\TerritorialCouncilQualityEnum;
+use App\Repository\TerritorialCouncilRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Internal\Hydration\IterableResult;
+use Doctrine\ORM\QueryBuilder;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Oneshot command, can be deleted after execution.
+ */
+class AddTerritorialCouncilMemberInfoCommand extends Command
```

Comment:
we can delete it?

Reply:
No, because we need to initialize "Referent" and "LRE manager" Territorial Council members.
I have just changed the name to `AddTerritorialCouncilMembershipCommand `.